plugins/generator-1.19.4/forge-1.19.4/procedures/team_get_entity_team.java.ftl

```diff
@@ -1,2 +1,2 @@
-(${input$entity} instanceof LivingEntity _teamEnt${cbi} && _teamEnt${cbi}.level.getScoreboard().getPlayersTeam(_teamEnt${cbi}.getStringUUID()) != null ?
-	_teamEnt${cbi}.level.getScoreboard().getPlayersTeam(_teamEnt${cbi}.getStringUUID()).getName() : "")
\ No newline at end of file
+(${input$entity} instanceof LivingEntity _teamEnt && _teamEnt.level.getScoreboard().getPlayersTeam(_teamEnt.getStringUUID()) != null ?

```

Comment:
What blocks do need this index then as I've seen other output blocks use it for pattern variables too?

Reply:
Blocks that can appear in || or &&